src/west/cmd/project.py

```diff
@@ -493,6 +512,17 @@ def _verify_repo(project):
                       "'git rev-parse --show-cdup'")
 
 
+def _branches(project):
+    # Returns a sorted list of all local branches in 'project'
```

Comment:
Is this really sorted?

Reply:
Yep, because `git for-each-ref` returns them sorted. Will add a clarifying comment.

Reply:
Thanks for explaining.

Reply:
Done